bogenliga/src/app/modules/user/services/login-data-provider.service.ts

```diff
@@ -46,6 +46,9 @@ export class LoginDataProviderService extends DataProviderService {
    */
   public signIn(credentialsDO: CredentialsDO): Promise<LoginResult> {
 
+    if(credentialsDO == this.getDefaultUser()) {
+      this.signInDefaultUser();
+    }
```

Comment:
Damit landest du in einer Endlosschleife! du rufst aus signIn SignInDefaultUser auf wo du dann wieder signIn aufrufst!

Reply:
Diese Abfrage brauchts nicht